http_handlers.go

```diff
@@ -302,6 +315,8 @@ func handleTrain(c echo.Context) error {
 
 	go func(args trainArgs) { ch <- args }(targs)
 
+	app.cache.Delete(langCode, targs.Pattern)
```

Comment:
why deleting pattern from cache?

Reply:
@joicemjoseph The cache stores the Pattern (english), so it's the english pattern that needs to be removed